herddb-core/src/main/java/herddb/core/system/SyscolumnsTableManager.java

```diff
@@ -39,8 +39,8 @@
     private final static Table TABLE = Table
             .builder()
             .name("syscolumns")
-            .column("table_name", ColumnTypes.STRING)
-            .column("column_name", ColumnTypes.STRING)
+            .column("table_name", ColumnTypes.NOTNULL_STRING)
```

Comment:
This MAY break compatibily with older clients (which are not aware of NOT NULL columns).
I would defer this change on a new PR

Reply:
I reverted both *table_name* and *column_name* changes for this file.